### PR TITLE
Self-Hosted Release Notes for v0.7

### PR DESCRIPTION
**Pull request by barberscott**
PR for Release Notes for LS Docs for the v0.7 SH release.

### CODE REVIEW

File: versioned_docs/version-2.0/self_hosting/release_notes.mdx [hunk 1]
```diff
@@ -6,14 +6,68 @@ sidebar_position: 7
...
+- [Enhanced trace comparison view and savable custom trace filters](https://changelog.langchain.com/announcements/trace-comparison-view-saving-custom-trace-filters)
+- [Defining, validating and updating dataset schemas](https://changelog.langchain.com/announcements/define-validate-and-update-dataset-schemas-in-langsmith)
+- [Multiple annotators can review a run in LangSmith](https://changelog.langchain.com/announcements/multiple-annotators-can-review-a-run-in-langsmith)
+- [Support for filtering runs within the trace view](https://changelog.langchain.com/announcements/filtering-runs-within-the-trace-view)
+- [Enhanced key-value search](https://changelog.langchain.com/announcements/enhanced-key-value-search-matching-inputs-and-outputs)
+- [Webhook notifications for run rules](https://changelog.langchain.com/announcements/set-up-webhook-notifications-for-run-rules)
+- [Support for comparing multiple prompts and model configurations side-by-side in the LangSmith Playground](https://changelog.langchain.com/announcements/build-prompts-faster-and-compare-in-langsmith-playground)
+- Support for saving playground configs
+- Resource Tags
```
Comment (langchain-infra) [inline versioned_docs/version-2.0/self_hosting/release_notes.mdx:27]: do we have docs or tweets for these?

File: versioned_docs/version-2.0/self_hosting/release_notes.mdx [hunk 2]
```diff
@@ -6,14 +6,68 @@ sidebar_position: 7
...
+  - By default we will use:
+    - 16 CPU
+    - 64 GB RAM
+    - You will need a node that can fit 4 CPU/16 GB RAM
+  - To override these settings you can manually configure resources requests/limits yourself
+- Turned bundled `Redis` persistence on by default. If you are using the bundled version of `redis` you may need to recreate your `Redis` StatefulSet if you had not previously turned on persistence.
+- Updated `clickhouseMigration` command to wait for clickhouse initialization prior to running migrations.
+- Deprecation of `<domain>/api-hub url`. You can now use the `<domain>/api` url for all api interactions. This will be fully removed in the v0.8 release so make sure to migrate any apps using the previous url.
+- Health checks added to `queue` pods.
+- Updates to the `nginx` config.
```
Comment (langchain-infra) [inline versioned_docs/version-2.0/self_hosting/release_notes.mdx:48]: removed openai key config option as well

File: versioned_docs/version-2.0/self_hosting/release_notes.mdx [hunk 3]
```diff
@@ -6,14 +6,68 @@ sidebar_position: 7
...
+
+### Admin Changes
+
+- Added support for simple password authentication (link to Simple auth config docs).
```
Comment (langchain-infra) [inline versioned_docs/version-2.0/self_hosting/release_notes.mdx:52]: think needs link?

File: versioned_docs/version-2.0/self_hosting/release_notes.mdx (hunk 1 above)
Comment (barberscott) [reply versioned_docs/version-2.0/self_hosting/release_notes.mdx]: Fixed.

**Comment by barberscott** [comment]
Resolved comments.  Ready for review.

**Review by langchain-infra — APPROVED** [review]

**Pull request merged**